lib/cmd_tool_isOneIp.go

```diff
@@ -45,24 +49,6 @@ func CmdToolIsOneIp(f CmdToolIsOneIpFlags, args []string, printHelp func()) erro
 	return GetInputFrom(args, true, true, op)
 }
 
-// CIDRContainsExactlyOneIP checks whether a CIDR contains exactly one IP
-func CIDRContainsExactlyOneIP(cidrStr string) bool {
-	_, ipnet, err := net.ParseCIDR(cidrStr)
-	if err != nil {
-		return false
-	}
-
-	if ipnet.IP.To4() != nil {
-		ipRange, _ := IPRangeStrFromCIDR(cidrStr)
-		return ipRange.Start == ipRange.End
-	} else if ipnet.IP.To16() != nil {
-		ipRange, _ := IP6RangeStrFromCIDR(cidrStr)
-		return ipRange.Start == ipRange.End
-	}
-
-	return false
-}
-
 func ipRangeContainsExactlyOneIP(ipRangeStr string) bool {
```

Comment:
`parsePrefix` only parses from CIDR notation so `ipRangeContainsExactlyOneIP` had to be kept